quinn-proto/src/connection/streams/state.rs

```diff
@@ -865,6 +858,17 @@ impl StreamsState {
             self.send_streams -= 1;
         }
     }
+
+    pub(super) fn max_send_data(&self, id: StreamId) -> VarInt {
```

Comment:
We should call this `initial_max_...` to avoid confusion with live flow control data.